src/wp-admin/theme-install.php

```diff
@@ -110,8 +110,8 @@
 $help_overview =
 	'<p>' . sprintf(
 		/* translators: %s: Theme Directory URL. */
-		__( 'You can find additional themes for your site by using the Theme Browser/Installer on this screen, which will display themes from the <a href="%s">ClassicPress Theme Directory</a>. These themes are designed and developed by third parties, are available free of charge, and are compatible with the license WordPress uses.' ),
-		__( 'https://wordpress.org/themes/' )
+		__( 'You can find additional themes for your site by using the Theme Browser/Installer on this screen, which will display themes from the <a href="%s">ClassicPress Theme Directory</a>. These themes are designed and developed by third parties, are available free of charge, and are compatible with the license ClassicPress uses.' ),
```

Comment:
Similar to the Plugin page, this contextual help shows while browsing WordPress repository themes so it should all say WordPress.